stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/typing/viewmodel/TypingIndicatorViewModel.kt

```diff
@@ -35,12 +35,4 @@ public class TypingIndicatorViewModel(
         channelState.typing.map { typingEvent ->
             typingEvent.users
         }.asLiveData()
-
-    private companion object {
-
-        /**
-         * The default limit for messages count in requests.
-         */
-        private const val DEFAULT_MESSAGE_LIMIT: Int = 30
```

Comment:
Also - most of these functions used this as the default limit and now it's `0`, any reason why this change?

Reply:
The reason why it's a 0 is because there is no benefit of message preloading here, it's just unnecessary because other than `MessageListViewViewModel`, our VMs do not benefit from having messages stored.

The reason why it's not just left empty is because @adasiewiczr  removed the default argument particularly for this reason of people leaving it mistakenly empty and pre loading messages when they should not be doing so.